ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -1397,6 +1400,111 @@ bfd_multihop:
   multiplier: 3
 ```
 
+## Role Enchancements for Super Spine Support
+
+The enchancement listed below are required to support bigger deployments with super-spines (5 stage CLOS).
+5 stage CLOS fabric can be represented as multiple leaf-spine structures (called PODs - Point of Delivery) interconnected by super-spines.
+The logic to deploy every leaf-spine POD fabric remains unchanged. The enchancement only adds logic required to provision spine-to-super-spine fabric.
+Super-spines can be deployed as a single plane (typically chassis switches) or multiple planes.
+Current AVD release supports single plane deployment only.
+
+Only BGP underlay is supported for super-spine deployment. Spines in every POD must have unique AS per POD.
+
+### Inventory Structure
+
+The inventory must have a dedicated group for super-spines and every leaf-spine POD. Example:
+
+```yaml
+all:
+  children:
+    < DC-group-name >:
+      children:
+        < Super Spines group name >:
+          hosts:
+            < super-spine name >:
+              ansible_host: < management IP >
+            < super-spine name >:
+              ansible_host: < management IP >
+            ...
+        < DC POD 1 group name >:
+          children:
+            < spines group >:
+              <-- omitted -->
+            < leaf group >:
+              <-- omitted -->
+```
+
+### Additional Variables Required For Super Spine Deployment
+
+Defaults:
+
+```yaml
+max_spine_to_super_spine_links: 1  # number of parallel links between spines and super-spines
+```
+
+Assigned to the DC group:
+
+```yaml
+dc_name: DC1  # data center fabric name
+
+max_super_spines: 4  # maximum number of super-spines, changing this parameter affects address allocation
+
+super_spine:
+  platform: vEOS-LAB  # super-spine platform
+  bgp_as: super-spine BGP AS
+  spine_interfaces:  # super-spine interfaces to spines
+  # interface list for every POD will be walked sequentially
+  # taking `max_spine_to_super_spine_links` into account
+  # for example: spine1, spine2, spine3, ... or spine1, spine1, spine2, spine2, etc.
+  - ['Ethernet1', 'Ethernet2', 'Ethernet3', 'Ethernet4']  # interfaces to POD1
+  - ['Ethernet1', 'Ethernet2', 'Ethernet3', 'Ethernet4']  # interfaces to POD2
```

Comment:
I think it would be more clear if the example here was:
  - ['Ethernet1', 'Ethernet1', 'Ethernet1', 'Ethernet1']  # interfaces to POD1
  - ['Ethernet2', 'Ethernet2', 'Ethernet2', 'Ethernet2']  # interfaces to POD2

Reply:
Well I think I just misunderstood this, so the example is correct, but I think the variable name should be super_spine_interfaces to avoid ambiguity.

Reply:
Connectivity should be defined from the child device, in case the spine. The data model should be similar to l3leaf to spine.